frontend/src/components/NavigationBar.tsx

```diff
@@ -0,0 +1,48 @@
+import { NavLink } from "react-router-dom";
+import { Container, List, ListItem } from "@mui/material";
+
+import { NavigationBarRoutes } from "src/routes/nav.routes";
+
+import LogoImage from "../assets/images/Logo/Logo.png";
+
+import { DrawerButton } from "./DrawerButton";
+
+export const NavigationBar = () => {
+  return (
+    <Container disableGutters sx={{ height: "100vh" }}>
+      <Container
+        component="img"
+        src={LogoImage}
+        sx={{ my: "10%", width: "80%" }}
+      />
```

Comment:
nit: We should use a minified logo at this place, perhaps we can replace it when we create the website favicon.